cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/internal/documentstore/ForceInsertItem.java

```diff
@@ -30,7 +30,7 @@
 
     /**
      * @param rev                   A {@code DocumentRevision} containing the information for a
-     *                              revision from a remote datastore.
+     *                              revision from a remote DocumentStore.
```

Comment:
Something like `revision from a remote Couch or Cloudant instance` should work? I just don't want to use datastore if it might cause confusion with the transition from datastore -> documentStore.